### PR TITLE
Add getCampaignInfo, update minimum hours, add call to remove inactiv…

### PR DESCRIPTION
**Pull request by 1szheng**
Add `getCampaignInfo()` in Campaign.sol
Update minimum hours to 24hrs
Add call from Campaign.sol's `withdraw` to remove inactive campaigns in campaignFactory

### CODE REVIEW

**Comment by froststein** [comment]
LGTM

**Review by froststein — APPROVED** [review]
LGTM

File: contracts/Campaign.sol [hunk 1]
```diff
@@ -12,9 +12,10 @@ contract Campaign {
     uint256 commissionBP = 1000; // 10% == (1000 / 10,000) basis points
     uint256 basispoints = 10000;
 
-    event donationMade (address donor, uint256 donatedAmt);
-    event hasWithdrawn (address from, uint256 totalDonationAmt);
-    event hasRefunded (address donor, uint256 refundedAmt);
+    event campaignInfo(address owner, string status, uint256 endDatetime, uint256 totalDonated);
```
Comment (TheSpaceCuber) [inline contracts/Campaign.sol]: Suggest to be more descriptive of an event name.
```suggestion
    event campaignInfoRetrieved(address owner, string status, uint256 endDatetime, uint256 totalDonated);
```

File: contracts/Campaign.sol [hunk 2]
```diff
@@ -73,6 +74,19 @@ contract Campaign {
...
+        uint256 statusInt = uint256(IAMContract.getStatus(owner));
+        string memory status;
+        if (statusInt == 1) {
+            status = "Verified";
+        } else if (statusInt == 2) {
+            status = "Locked";
+        } else {
+            status = "Distrust";
+        }
+        emit campaignInfo(owner, status, endDatetime, totalDonated);
```
Comment (TheSpaceCuber) [inline contracts/Campaign.sol]:
```suggestion
        emit campaignInfoRetrieved(owner, status, endDatetime, totalDonated);
```

**Review by TheSpaceCuber — APPROVED** [review]

**Review by YitHien — APPROVED** [review]
LGTM

**Pull request merged**